### ISSUE TITLE
View mode not accessible from user or comment template files

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#5280]
**Description of the bug**

With nodes and terms you can access the `$view_mode` variable from the template file, but it does not seem to be available for user profiles. (Or potentially comments based on a little research I've done).

**Comment by indigoxela** [comment]
Yes, the view_mode currently isn't available in both theme files, comment.tpl.php and user-profile.tpl.php.

As both are fieldable entities, the access to these variables might come handy for theming.

One thought: can the "bundle" in function template_preprocess_user_profile() ever be anything else than "user"? :wink: So maybe it's redundant there. For comments this makes sense.

One request: the additional variables should also get mentioned in both template doc blocks. Otherwise hardy any themer will be aware of them.

**Comment by laryn** [comment]
Thanks @indigoxela -- I just found the typos in each template file but didn't think beyond fixing them. I think you are right about removing `bundle` from the user profile. I'll make a few tweaks here and update the PR.

**Comment by indigoxela** [comment]
@laryn I left another question on your PR.

**Comment by laryn** [comment]
@indigoxela The PR has been adjusted based on your comment.

**Comment by indigoxela** [comment]
> The PR has been adjusted

Great, this all looks good now! @laryn One final request: can you rebase your PR? It's 46 commits behind, which is a lot.

**Comment by laryn** [comment]
@indigoxela I pulled those commits in.

**Comment by indigoxela** [comment]
@laryn Many thanks. I did some local testing and can confirm, that $bundle and $view_mode are available/usable now. Tested with comment.tpl.php.

This was probably just an oversight, as the variables have been created in both functions, template_preprocess_comment and template_preprocess_user_profile, but then they haven't been used (not handed over to the template). As the change is really small, I take the liberty to RTBC in one step.

But we still need someone to set the milestone. And an additional pair of eyes is never wrong. :wink:
